src/pages/waiting-room/waiting-room.module.ts

```diff
@@ -5,7 +5,10 @@ import { EffectsModule } from '@ngrx/effects';
 import { WaitingRoomAnalyticsEffects } from './waiting-room.analytics.effects';
 import { AnalyticsProvider } from '../../providers/analytics/analytics';
 import { PreTestDeclarationsModule } from '../../modules/test/pre-test-declarations/pre-test-declarations.module';
+import { ComponentsModule } from '../../components/components.module';
 
+// import { MesSignaturePadComponent } from '../../components/mes-signature-pad/mes-signature-pad';
+// import { SignaturePadModule } from 'angular2-signaturepad';
```

Comment:
Comments - can we remove them if not needed

Reply:
my bad.. well spotted